### ISSUE TITLE
[ENHANCEMENT] Support for the different versions that Essential supports.

### ISSUE THREAD

**Issue by KTrain5169** [usevital/notessential#43]
Our site is heavily based around Essential alternatives for modern versions of Minecraft. As such, people on older versions can't use our site much. I think adding this feature can help these people.

Checklist:
- [ ] Site change: Add options (maybe through buttons) on /alternatives that allows the user to switch between the different version of Minecraft that Essential supports.
- [ ] Site change: Add a special section (maybe represented by images?) that denote what loader the mod works for and what versions of Minecraft does that mod support on that loader.
- [ ] Crowdin: Update with new strings

**Comment by blryface** [comment]
This seems like a *lot* of work to do with how many versions Essential supports. It would make more sense if @intergrav got around to making Vital versions for them, based off of our list

**Comment by KTrain5169** [comment]
> This seems like a _lot_ of work to do with how many versions Essential supports. It would make more sense if @intergrav got around to making Vital versions for them, based off of our list

We don't have to actually implement a button and slider system for this or whatever, it can just be new notes for what version and corresponding loader the mod supports

It can be that simple, but it'd honestly make a lot of difference

**Comment by worldwidepixel** [comment]
It's super nice only having one issue/PR open

**Comment by KTrain5169** [comment]
Unfortunately this issue will be delayed because🤷‍♂️

**Comment by worldwidepixel** [comment]
Yeah I'm not doing this for the forsesable future lol

**Comment by blryface** [comment]
i think it should be closed as not planned

**Comment by blryface** [comment]
i don't think we would even be able to figure out a way to show them in a good way lol

**Comment by blryface** [comment]
guess
![image](https://github.com/user-attachments/assets/9196efc9-2eec-48db-8b3c-b9dba237e2e3)

**Comment by worldwidepixel** [comment]
no idea

**Comment by KTrain5169** [comment]
Not planned, maybe someday

**Comment by blryface** [comment]
We should definitely look into this again as we also redo the `Why` section.

Perhaps we should have something like Modrinth's project type selector, but instead for many versions of the game

Maybe we should also have a `Vital Legacy` modpack that has versions of the Vital modpack for older versions of Minecraft too? It wouldn't need to be maintained a lot, so even if it takes a lot of effort we won't need to redo it much if ever

**Comment by blryface** [comment]
The checklist on KTrain's comment seems alright so I won't edit it
